doc/extending.md

```diff
@@ -61,3 +61,28 @@ can run the templating manually:
 ```
 ./baselayer/tools/fill_conf_values.py --config="config.yaml" static/js/component.jsx.template
 ```
+
+## Adding extrernal services
+
+If you want to add external services to your application,
```

Comment:
I think we should explain what *is* an external service before saying how to add one. Explaining the concept, etc.

Reply:
:+1: I can't see higher up in that file, but we should check that the concept of a *service* is well defined. Also make very clear that external services are identical to services, they just live in remote repos.